apps/mux/src/index.tsx

```diff
@@ -383,11 +458,27 @@ export class App extends React.Component<AppProps, AppState> {
         );
       }
 
-      if (this.state.value.ready) {
+      if (
+        this.state.value.ready &&
+        this.state.playbackUrl &&
+        this.state.posterUrl
+      ) {
         return (
           <div>
+            {this.isUsingSigned() && (
+              <Note>
+                Note: this mux asset is using a{' '}
+                <TextLink
+                  href="https://docs.mux.com/docs/headless-cms-contentful#advanced-signed-urls"
+                  target="_blank"
```

Comment:
Could you please add `rel="noopener noreferrer"`?

Reply:
done!